gmr/__init__.py

```diff
@@ -20,11 +20,19 @@
 if not __GMR_SETUP__:
     from . import gmm, mvn, utils
 
+    import warnings
+
     __all__ = ["gmm", "mvn", "utils"]
 
     from .mvn import MVN, plot_error_ellipse
     from .gmm import (GMM, plot_error_ellipses, kmeansplusplus_initialization,
                       covariance_initialization)
 
+    try:
```

Comment:
I would try to avoid that you see this warning every time when you import gmr in your code. I would add sklearn to `__all__`. Then I would suggest to do something like:

```python
try:
    import sklearn
except ImportError:
    raise ImportError("Install scikit-learn (e.g. pip install scikit-learn) to use this extension.")

Reply:
Moved it so it can be called as:

```python
from gmr.sklearn import GaussianMixtureRegressor
```

Do you think this is ok?

Reply:
It's always easier to add an additional import in the __init__.py than to remove it with deprecation and stuff. If we think that we need an additional import, we can just add it later.